test/powershell/Modules/Microsoft.PowerShell.Utility/Select-Object.Tests.ps1

```diff
@@ -6,249 +6,227 @@ Add-TestDynamicType
 
 Describe "Select-Object" -Tags "CI" {
     BeforeEach {
-	$dirObject  = GetFileMock
-	$TestLength = 3
+        $dirObject = GetFileMock
+        $TestLength = 3
     }
 
     It "Handle piped input without error" {
-	{ $dirObject | Select-Object } | Should -Not -Throw
+        { $dirObject | Select-Object } | Should -Not -Throw
     }
 
     It "Should treat input as a single object with the inputObject parameter" {
-	$result   = $(Select-Object -InputObject $dirObject -Last $TestLength).Length
-	$expected = $dirObject.Length
-
-	$result | Should -Be $expected
-    }
-
-    It "Should be able to use the alias" {
-	{ $dirObject | select } | Should -Not -Throw
+        $result = $(Select-Object -InputObject $dirObject -Last $TestLength).Length
+        $expected = $dirObject.Length
+        { $dirObject | Select-Object } | Should -Not -Throw
```

Comment:
`Should -Not -Throw` is not good test. 
Why do we remove one test below? And why do we need `$expected` above? Why do we change the tests at all?